### DISCUSSION TITLE
How can I render my documents out of outline?

### DISCUSSION THREAD

**Discussion by Monstermash28425R1** [outline/outline#5192] [General]
How can I accurately render the Markdown documents that I obtain through the API from my self-hosted instance of Outline? I have successfully retrieved the text in Markdown format, but I am looking for a CSS file or other solution to ensure that the text is displayed in the same way as in the Outline editor. This is not an error, as the Markdown text is retrieved correctly, but I need a way to apply the Outline user interface style to the rendered documents in my application.

![imagen](https://user-images.githubusercontent.com/28737271/231519530-e9559b71-b340-492b-af7c-7c2e058b8a7d.png)

this is how it looks like right now using a github-style markdown css

**Comment by tommoor** [comment]
You should probably strip the lines that are a single `\`, these represent an empty line but most Markdown renderers won't respect that.

For CSS it's up to you, Outline doesn't style Markdown directly so there is no stylesheet to pull.
